node/repo/interface.go

```diff
@@ -64,7 +64,7 @@ type LockedRepo interface {
 	// The supplied context must only be used to initialize the blockstore.
 	// The implementation should not retain the context for usage throughout
 	// the lifecycle.
-	Blockstore(ctx context.Context, domain BlockstoreDomain) (blockstore.Blockstore, error)
+	Blockstore(ctx context.Context, domain BlockstoreDomain) (blockstore.Blockstore, func() error, error)
```

Comment:
would be good to document what the returned `func` is in the comment, I had to go to the bottom of the only function that implements it to find the `closer`

Reply:
or type it, `type CloserFn func() error`